packages/ui/src/primitives/tailwind/Select/index.tsx

```diff
@@ -462,6 +458,10 @@ const Select = ({
                 }
                 setTouchedMoved(false)
               }}
+              onPointerUp={(e) => {
+                e.stopPropagation()
+                e.preventDefault()
+              }}
```

Comment:
imo, this is the only change that we were missing

(events were being propagated to clickawaylistener)